docs/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Getting Started
+nav: 0.1
+---
+Welcome to the Jotai Getting Started tutorial!
+This tutorial will introduce you to Jotai and teach you how to start using it correctly.
+
+For this tutorial, we assume that you're using Jotai with React. The example is based on [a Create-React-App folder structure](https://create-react-app.dev/docs/folder-structure) so the first thing you have to do is:
+
+`npx create-react-app my-app`
+
+## Installation
+To install the latest stable version of Jotai, run the following command:
+
+`npm install jotai` or `yarn add jotai`
+
+## Provider
+
+Provider is to provide a state for component sub tree.
+This works just like the normal React Context.
+A good place to put this is in your root component:
+
+```jsx
+import { Provider, atom, useAtom } from 'jotai'
+
+const App = () => (
+  <Provider>
+    <Input />
+    <CharCount />
+    <Uppercase />
+  </Provider>
+)
+
+export default App
+```
+
+We'll implement the components mentioned above in the following section.
```

Comment:
We should mention what we are going to develop in the following.

Reply:
I didn't understand

Reply:
I didn't understand where something is "mentioned above".

So, it would be nice to tell briefly to the readers about what we will be developing in the getting started section, before diving into the code.